backend/src/docker-compose-dep.yml

```diff
@@ -8,6 +8,7 @@ services:
       - POSTGRES_PASSWORD=docker
       - POSTGRES_USER=docker
       - POSTGRES_DB=postgres
+      - PGDATA=/tmp
```

Comment:
have you tried this on linux/mac?

Reply:
Nope, I have Windows 10 operating system maybe someone else can try.

Reply:
I could try this on WSL, but I'm not sure that replicates a Linux environment 100%.
When I had to work with the database, I did these changes myself but haven't committed them. We should first make sure it doesn't break other stuff(especially on the prod env), but yeah, it would be nice to have Windows support as well.